packages/assets-controllers/src/NftDetectionController.ts

```diff
@@ -115,9 +115,8 @@ export interface ApiNftCreator {
  *
  * NftDetection configuration
  * @property interval - Polling interval used to fetch new token rates
- * @property networkType - Network type ID as per net_version
+ * @property chainId - Current chain ID
  * @property selectedAddress - Vault selected address
- * @property tokens - List of tokens associated with the active vault
```

Comment:
bad copy pasta clean up?

Reply:
right. there is no `tokens` property on this interface